pkg/config/system_probe.go

```diff
@@ -164,16 +164,21 @@ func InitSystemProbeConfig(cfg Config) {
 	cfg.BindEnvAndSetDefault(join(spNS, "source_excludes"), map[string][]string{})
 	cfg.BindEnvAndSetDefault(join(spNS, "dest_excludes"), map[string][]string{})
 
-	// network_config namespace only
-	cfg.BindEnv(join(netNS, "enable_http_monitoring"), "DD_SYSTEM_PROBE_NETWORK_ENABLE_HTTP_MONITORING")
-	cfg.BindEnv(join(netNS, "enable_https_monitoring"), "DD_SYSTEM_PROBE_NETWORK_ENABLE_HTTPS_MONITORING")
+	// service monitoring namespace only
+	cfg.BindEnv(join(smNS, "enable_http_monitoring"))
+	// Setting join(smNS, "enable_http_monitoring") as alias to join(netNS, "enable_http_monitoring"), will allow us to
+	// either set the field in the configuration file in either of the options.
+	cfg.SetAliasAndEnv(join(netNS, "enable_http_monitoring"), join(smNS, "enable_http_monitoring"), "DD_SYSTEM_PROBE_NETWORK_ENABLE_HTTP_MONITORING")
```

Comment:
do we want to keep the env variable with `NETWORK` inside of it or should we introduce a new variable as well and keep the old one for backward compatibility?

Reply:
I'm keeping it for backward compatibility, we have the new way of `DD_SERVICE_MONITORING_CONFIG_ENABLE_HTTP_MONITORING`